zkserver/zkserver.go

```diff
@@ -659,8 +700,7 @@ func (z *ZKS) listen() error {
 				continue
 			}
 
-			conn.(*net.TCPConn).SetKeepAlive(true)
-			conn.(*net.TCPConn).SetKeepAlivePeriod(time.Second)
+			conn.(*net.TCPConn).SetKeepAlive(false)
```

Comment:
recommend just removing altogether

Reply:
I like it being explicit and I don't know what the OS is doing underneath.